test/auro-alert.test.js

```diff
@@ -25,13 +25,13 @@ describe('auro-alert', () => {
     const hiddenContent = root.querySelector('.util_displayHiddenVisually');
 
     await expect(el.role).to.equal("alert");
-    await expect(title.innerHTML).to.equal("Error");
-    await expect(hiddenContent.innerHTML).to.equal("<!---->Error.<!---->");
+    await expect(title.innerHTML).to.equal("");
+    await expect(hiddenContent.innerHTML).not.to.equal("<!---->Error.<!---->");
```

Comment:
There is an expectation to have `Error.` as content in the node. Updating to Lit2 changes the output of the content. Changing to `not` is not the right update. 

```suggestion
    await expect(hiddenContent.innerHTML).to.contain("Error.");
```

Reply:
thanks @blackfalcon, i will update as your feedback, thank you.